tools/swift-ide-test/swift-ide-test.cpp

```diff
@@ -691,6 +691,11 @@ static llvm::cl::opt<bool>
                        llvm::cl::desc("Disable ObjC interop."),
                        llvm::cl::cat(Category), llvm::cl::init(false));
 
+static llvm::cl::opt<bool>
```

Comment:
Do you have IDE tests that require this flag?

Reply:
Yes -- see the test access-specifiers-module-interface.swift added in this PR.

Or do you mean a test that actually exercises features used in an IDE? In that case, no. My understanding, however, is that %target-swift-ide-test is the standard way of creating module interface tests?

Reply:
Ah, good. My eyes scanned over it - not used to seeing swift-ide-test being used to dump interfaces outside of the IDE test subdirectory...